File: src/Omnia.ProcessManagement.Models/ProcessTypes/PublishingApprovalSettings/PersonPropertyPublishingApprovalSettings.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.ProcessTypes.PublishingApprovalSettings
{
    public class PersonPropertyPublishingApprovalSettings : PublishingApprovalSettings
    {
        public override PublishingApprovalSettingsTypes Type
        {
            get
            {
                return PublishingApprovalSettingsTypes.PersonProperty;
            }
        }

        public Guid PersonEnterprisePropertyDefinitionId { get; set; }
    }
}

File: src/Omnia.ProcessManagement.Web/client/fx/models/data/processtypes/publishingapprovalsettings/PersonPropertyPublishingApprovalSettings.ts
import { PublishingApprovalSettings, PublishingApprovalSettingsTypes } from '.';
import {  GuidValue } from '@omnia/fx-models';

export interface PersonPropertyPublishingApprovalSettings extends PublishingApprovalSettings {
    type: PublishingApprovalSettingsTypes.PersonProperty;
    personEnterprisePropertyDefinitionId: GuidValue;
}